content/en/blog/_posts/2019-06-19-Future-of-CRDs-Structural-Schemas.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: blog
+title: "Future of CRDs: Structural Schemas"
+date: 2019-06-18
```

Comment:
I believe we need this updated to 2019-06-19 as well

Reply:
done